packages/shared/src/components/Feed.tsx

```diff
@@ -67,6 +68,11 @@ const SharePostModal = dynamic(
     import(/* webpackChunkName: "sharePostModal" */ './modals/SharePostModal'),
 );
 
+const ActivateCompanionModal = dynamic(
+  () =>
+    import(/* webpackChunkName: "CompanionModal" */ './modals/CompanionModal'),
+);
+
```

Comment:
Ideally, we would not have this duplicated usage of the `CompanionModal` in both `Feed` and `PostContent`, we could place it higher in the tree, but then we would need to add a `Context` to manage its state -- I am open to suggestions if the team prefer this

Reply:
Good point. Though the state updates might cause re-renders. We currently have not been using `memo`, so we are not yet optimized for unexpected re-renders.

Reply:
done with https://github.com/dailydotdev/apps/pull/2039/commits/7fa0a89f8e9e8b041e43e7a0da91a27277b5db0c